civiform_config.example.sh

```diff
@@ -328,3 +336,47 @@ export AD_GROUPS_ATTRIBUTE_NAME="group"
 # The ADFS group name for specifying CiviForm admins. If using Azure AD this is
 # the group's object ID
 export ADFS_ADMIN_GROUP=""
+
+# REQUIRED if CIVIFORM_ADMIN_IDP="generic-oidc-admin"
+# The name of the OIDC provider. Must be URL-safe.
+#
+# Gets appended to the auth callback URL, as in:
+#   ${BASE_URL}/callback/${ADMIN_OIDC_PROVIDER_NAME}"
+#
+# The callback URL may need to be provisioned in the auth provider.
+export ADMIN_OIDC_PROVIDER_NAME=""
```

Comment:
**thought:** Thoughts on commenting out, since this is not always required? And same with others. I see a mix above, so I'm good with either, but a thought.

Reply:
Good idea. Done.